duckduckgo/duckduckgo-privacy-extension
# Add "atbd" param to exti call, unit test urls, and expected output

Reviewer: @andrey-p

Depends on: Internal Changes

# Description: #
Adds ability to parse additional url param (atbd) from success page

# Steps to test this PR: #

1. Visit https://duckduckgo.com?natb=v123-1zz&cp=atbzz&atbd=4
2. Install extension
3. Ajax call to /exti/ should include atbd=4 (See Kibana)

# Automated tests: #

# Reviewer Checklist: #


# PR Author Checklist: #
